faker/providers/address/de_CH/__init__.py

```diff
@@ -0,0 +1,44 @@
+from .. import Provider as AddressProvider
+
+
+class Provider(AddressProvider):
+    city_formats = ('{{canton_name}}',)
+    building_number_formats = ('%', '%#', '%#', '%#', '%##')
+    street_suffixes = ['strasse']
+    street_name_formats = ('{{last_name}}{{street_suffix}}', )
+    street_address_formats = ('{{street_name}} {{building_number}}', )
+    address_formats = ('{{street_address}}\n{{postcode}} {{city}}', )
+    postcode_formats = ('1###', '2###', '3###', '4###', '5###', '6###', '7###',
+                        '8###', '9###')
+
+    cantons = (('AG', 'Aargau'), ('AI', 'Appenzell Innerrhoden'),
+               ('AR', 'Appenzell Ausserrhoden'), ('BE', 'Bern'),
+               ('BL', 'Basel-Landschaft'), ('BS', 'Basel-Stadt'), ('FR', 'Freiburg'),
+               ('GE', 'Genf'), ('GL', 'Glarus'), ('GR', 'Graubünden'), ('JU', 'Jura'),
+               ('LU', 'Luzern'), ('NE', 'Neuenburg'), ('NW', 'Nidwalden'), ('OW', 'Obwalden'),
+               ('SG', 'St. Gallen'), ('SH', 'Schaffhausen'), ('SO', 'Solothurn'),
+               ('SZ', 'Schwyz'), ('TG', 'Thurgau'), ('TI', 'Tessin'), ('UR', 'Uri'),
+               ('VD', 'Waadt'), ('VS', 'Wallis'), ('ZG', 'Zug'), ('ZH', 'Zürich'))
+
+    def canton(self):
+        """
+        Randomly returns a swiss canton ('Abbreviated', 'Name').
+        :example ('ZH', 'Zürich')
+        """
+        return self.random_element(self.cantons)
+
+    def administrative_unit(self):
```

Comment:
Can we add an alias from state() to administrative_unit()? Not adding the state breaks backwards compatibility.

@fcurella @andre-dasilva

Reply:
```
  File "/usr/local/lib/python3.9/site-packages/faker/proxy.py", line 105, in __getattr__
    return getattr(self._factories[0], attr)
AttributeError: 'Generator' object has no attribute 'state'
```

Reply:
@m-aciek thats good for me.